### PR TITLE
Quickstart fixes

### PR DESCRIPTION
**Pull request by plotnick**

### CODE REVIEW

File: docs/getting-started/quickstart.rst [hunk 1]
```diff
@@ -404,7 +404,7 @@ We'll first start our server...
     .. code-block:: console
 
         $ javac -cp oso-0.2.5.jar:. Server.java
-        $ java Server
+        $ java -cp oso-0.2.5.jar:. Server
```
Comment (leina05) [inline docs/getting-started/quickstart.rst:407]: Someone had a way of replacing text, would be cool to do that here

Comment (leina05) [reply docs/getting-started/quickstart.rst:407]: Sorry that's Sam's currently open PR, ignore

Comment (plotnick) [reply docs/getting-started/quickstart.rst:407]: No, you're right. Just rebased on top of it.

File: docs/getting-started/quickstart.rst [hunk 2]
```diff
@@ -213,7 +213,7 @@ start a REPL session and follow along:
 
     .. code-block:: irb
 
-      irb(main):005:0> OSO.allowed?(actor: "alice", action: "GET", resource: "expense")
+      irb(main):005:0> OSO.allowed?(actor: alice, action: "GET", resource: "expense")
```
Comment (gj) [inline docs/getting-started/quickstart.rst:216]: super nit: all of these `irb` line numbers should be sequential

currently it goes 001 002 003 004 005 006 005 005 006

**Pull request merged**